src/gui/accountsettings.cpp

```diff
@@ -385,8 +385,10 @@ void AccountSettings::slotEditCurrentLocalIgnoredFiles()
 
 void AccountSettings::openIgnoredFilesDialog(const QString & absFolderPath)
 {
-    Q_ASSERT(absFolderPath.startsWith('/'));
-    Q_ASSERT(absFolderPath.endsWith('/'));
+#if !defined QT_NO_DEBUG
```

Comment:
@mgallien As far as I remember, `Q_ASSERT` is empty in release build, so, this can be shortened to:
`Q_ASSERT(QFileInfo(absFolderPath).isAbsolute())`

Reply:
Thanks, much better now.